x-pack/plugins/observability_solution/entity_manager/server/lib/entities/transform/generate_metadata_aggregations.ts

```diff
@@ -48,13 +70,49 @@ export function generateLatestMetadataAggregations(definition: EntityDefinition)
         aggs: {
           data: {
             terms: {
-              field: metadata.destination ?? metadata.source,
-              size: metadata.limit ?? ENTITY_DEFAULT_METADATA_LIMIT,
+              field: metadata.destination,
+              size: metadata.aggregation.limit,
+            },
+          },
+        },
+      };
+    } else if (metadata.aggregation.type === 'top_value') {
+      agg = {
+        filter: {
+          bool: {
+            must: [
+              {
+                range: {
+                  '@timestamp': {
+                    gte: `now-${offsetInSeconds}s`,
```

Comment:
This particular range filter for this agg doesn't quite fit some of the use cases for this type of aggregation. My understanding of this filter is effectively "only pull from the history entities since the last run of the history transform". Instead, for this particular aggregation on the latest transform, we'd want it to be configurable. 

Should we introduce a parameter at the aggregation level for this in the entity definition?

Something like a lookback parameter:

```
 metadata: [
    { source: 'host.name', aggregation: { lookback: '24h', type: 'top_value', sort: { '@timestamp': 'desc' } } }
  ]
```

Which would result in this line being:

```
gte: `now-${metadata.aggregation.lookback ?? offsetInSeconds + 's'}`
```

Reply:
That makes sense I'll update

Reply:
added in https://github.com/elastic/kibana/pull/188243/commits/517f2216391c0f51d9195436014528ff205bdd17